src/pixelmapFeature.js

```diff
@@ -0,0 +1,451 @@
+ *   transparent.  If there is more data than there are indices, it is ignored.
+ * @param {Object|Function} [position] Position of the image.  Default is
+ *   (data).  The position is an Object which specifies the corners of the
+ *   quad: ll, lr, ur, ul.  At least two opposite corners must be specified.
+ *   The corners do not have to physically correspond to the order specified,
+ *   but rather correspond to that part of the image map.  If a corner is
+ *   unspecified, it will use the x coordinate from one adjacent corner, the y
+ *   coordinate from the other adjacent corner, and the average z value of
+ *   those two corners.  For instance, if ul is unspecified, it is
+ *   {x: ll.x, y: ur.y}.  Note that each quad is rendered as a pair of
+ *   triangles: (ll, lr, ul) and (ur, ul, lr).  Nothing special is done for
+ *   quads that are not convex or quads that have substantially different
+ *   transformations for those two triangles.
+ * @returns {geo.pixelmapFeature}
+ */
+//////////////////////////////////////////////////////////////////////////////
+
+//////////////////////////////////////////////////////////////////////////////
+var pixelmapFeature = function (arg) {
+  'use strict';
+  if (!(this instanceof pixelmapFeature)) {
+    return new pixelmapFeature(arg);
+  }
+  arg = arg || {};
+  feature.call(this, arg);
+
+  ////////////////////////////////////////////////////////////////////////////
+  /**
+   * @private
+   */
+  ////////////////////////////////////////////////////////////////////////////
+  var m_this = this,
+      m_quadFeature,
+      m_srcImage,
+      m_info,
+      s_update = this._update,
+      s_init = this._init,
+      s_exit = this._exit;
+
+  ////////////////////////////////////////////////////////////////////////////
+  /**
+   * Get/Set position accessor
+   *
+   * @returns {geo.pixelmap}
+   */
+  ////////////////////////////////////////////////////////////////////////////
+  this.position = function (val) {
+    if (val === undefined) {
+      return m_this.style('position');
+    } else if (val !== m_this.style('position')) {
+      m_this.style('position', val);
+      m_this.dataTime().modified();
+      m_this.modified();
+    }
+    return m_this;
+  };
+
+  ////////////////////////////////////////////////////////////////////////////
+  /**
+   * Get/Set url accessor
+   *
+   * @returns {geo.pixelmap}
+   */
+  ////////////////////////////////////////////////////////////////////////////
+  this.url = function (val) {
+    if (val === undefined) {
+      return m_this.style('url');
+    } else if (val !== m_this.style('url')) {
+      m_srcImage = m_info = undefined;
+      m_this.style('url', val);
+      m_this.dataTime().modified();
+      m_this.modified();
+    }
+    return m_this;
+  };
+
+  ////////////////////////////////////////////////////////////////////////////
+  /**
+   * Get the maximum index value from the pixelmap.  This is a value present in
```

Comment:
'in' ends at column 79.  Do you really want to change to less than 80 line lengths?